src/main/java/edu/hm/hafner/coverage/CoverageParser.java

```diff
@@ -199,6 +203,18 @@ protected static int parseInteger(final String value) {
         }
     }
 
+    protected static Value createValue(final String currentType, final int covered, final int missed) {
+        if (VALUE_COMPLEXITY.equals(currentType)) {
+            return new Value(Metric.CYCLOMATIC_COMPLEXITY, covered + missed);
+        }
+        else {
```

Comment:
Moved this method from JacocoParser to parent class in order for GoParser to use the same method.